sw/device/silicon_creator/lib/sigverify_mod_exp_ibex_unittest.cc

```diff
@@ -195,7 +195,7 @@ TEST_P(ModExp, EncMsg) {
 TEST(ModExp, BadExp) {
   // Exponent = 0
   constexpr sigverify_rsa_key_t bad_key{};
-  sigverify_rsa_buffer_t empty;
+  sigverify_rsa_buffer_t empty = {0};
```

Comment:
nit: `sigverify_rsa_buffer_t empty{};` (just for consistency with the rest of the C++ code).

Reply:
Uh, I should have seen that -- it's only one line up. Thanks!